packaging/mingw-w64/configure-build-msys2-mingw.sh

```diff
@@ -10,15 +10,48 @@ cmake \
     -D RDKAFKA_BUILD_TESTS=ON \
     -D WITH_LIBDL=OFF \
     -D WITH_PLUGINS=OFF \
-    -D WITH_SASL=ON \
+    -D WITH_SASL=OFF \
     -D WITH_SSL=ON \
     -D WITH_ZLIB=OFF \
-    -D RDKAFKA_BUILD_STATIC=OFF \
-    -D CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=TRUE .
+    -D RDKAFKA_BUILD_STATIC=ON \
+    .
 
 $mingw64 mingw32-make
 $mingw64 mingw32-make install
 
+# Bundle all the static dependencies with the static lib we just built
+mkdir mergescratch
+pushd mergescratch
+cp /C/tools/msys64/mingw64/lib/libzstd.a ./
+cp /C/tools/msys64/mingw64/lib/libcrypto.a ./
+cp /C/tools/msys64/mingw64/lib/liblz4.a ./
+cp /C/tools/msys64/mingw64/lib/libssl.a ./
+cp ../src/librdkafka.a ./
+cp ../src-cpp/librdkafka++.a ./librdkafkapp.a
+ar -M << EOF
+create librdkafka_merge.a
```

Comment:
We call these bundled archives librdkafka-static.a in the *nix build, suggesting doing the same here.